CK3toEU4/Source/Mappers/ProvinceMapper/ProvinceMapper.cpp

```diff
@@ -86,12 +85,12 @@ std::map<std::string, std::shared_ptr<CK3::Title>> mappers::ProvinceMapper::getC
 					 "Province mapper error - requested ck3 province ID " + std::to_string(ck3ID) + " is invalid. We cannot find related county. Farewell.");
 			}
 		}
-		return toReturn;
 	}
 	else
 	{
-		throw std::runtime_error("Province mapper error - requested EU4 province ID " + std::to_string(eu4ProvinceNumber) + " does not exist in mappings.");
+		// We must not show fear here. Parent is likely iterating over all EU4 provinces. Play dead, they will go away.
```

Comment:
I must not fear. Fear is the mind-killer. Fear is the little death that brings total obliteration. I will face my fear. I will let it pass over me and through me. And when it has passed, I will turn my eye inward and see that only I remain.

Reply:
More practically, you can just drop the else block entirely, unless you think that will be confusing when you come back in six months or so.

Reply:
That's why I leave these blocks around - to make sure I recall considering them and deciding on no action.